.env

```diff
@@ -196,6 +203,24 @@ KEYCLOAK_ADMIN_PASSWORD=__CHANGE_ME_rTLDzTAFiFIQiHDm
 DEFAULT_ADMIN_USERNAME=phrasea-admin
 DEFAULT_ADMIN_PASSWORD=__CHANGE_ME_CAZ7B1ZP4os2kZPL
 
+LOCALISATION_ENABLED=true
+SUPPORTED_LOCALES=fr,en,es,pt
+DEFAULT_LOCALE=fr
+
+LOGIN_REGISTRATION_ALLOWED=false
+LOGIN_RESET_PASSWORD_ALLOWED=true
+LOGIN_REMEMBER_ME_ALLOWED=true
+LOGIN_WITH_EMAIL_ALLOWED=true
+LOGIN_VERIFY_EMAIL_ALLOWED=false
+LOGIN_EMAIL_AS_USERNAME=false
+LOGIN_EDIT_USERNAME=false
+
+SECURITY_DETECTION_BRUTE_FORCE_ENABLED=false
+
+SSO_SESSION_IDLE_TIMEOUT=1800
+CLIENT_SESSION_IDLE_TIMEOUT=1800
+OFFLINE_SESSION_IDLE_TIMEOUT=2592000
```

Comment:
Please see with @nmaillat what settings are really usefull and should be synced with HELM (we've discussed about it).
Also, prefix those env with `KC_REALM_`